examples/p4est_3d_dgsem/elixir_linearizedeuler_convergence.jl

```diff
@@ -0,0 +1,64 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linearized Euler equations
+
+equations = LinearizedEulerEquations3D(v_mean_global = (0.0, 0.0, 0.0), c_mean_global = 1.0,
+                                       rho_mean_global = 1.0)
+
+initial_condition = initial_condition_convergence_test
+
+solver = DGSEM(polydeg = 3, surface_flux = flux_hll)
+
+coordinates_min = (-1.0, -1.0, -1.0) # minimum coordinates (min(x), min(y), min(z))
+coordinates_max = (1.0, 1.0, 1.0) # maximum coordinates (max(x), max(y), max(z))
+
+# `initial_refinement_level` is provided here to allow for a 
+# convenient convergence test, see
+#  https://trixi-framework.github.io/Trixi.jl/stable/#Performing-a-convergence-analysis
+trees_per_dimension = (4, 4, 4)
+mesh = P4estMesh(trees_per_dimension, polydeg = 3,
```

Comment:
I chose `P4estMesh` here to test the functions involving normal vectors (for coverage).